README.adoc

```diff
@@ -312,39 +384,41 @@ In addition, a few endpoint tests have been included for you to test the basic f
 `inventory` and `system` services. If a test failure occurs, then you might have introduced a bug into
 the code.
 
-FaultToleranceTest.java
+FaultToleranceIT.java
 [source, java, linenums, role='code_column']
 ----
-include::finish/src/test/java/it/io/openliberty/guides/faulttolerance/FaultToleranceTest.java[tags=ft_testing;!javadoc]
+include::finish/src/test/java/it/io/openliberty/guides/faulttolerance/FaultToleranceIT.java[tags=ft_testing;!javadoc]
 ----
 // =================================================================================================
 // Running the tests
 // =================================================================================================
 
-[role='command']
-include::{common-includes}/mvnverify.adoc[]
+=== Running the tests
+
+Since you started Open Liberty in development mode at the start of the guide, press the `enter/return` key to run the tests. You will see the following output:
 
 [source, role="no_copy"]
 ----
 -------------------------------------------------------
  T E S T S
 -------------------------------------------------------
-Running it.io.openliberty.guides.system.SystemEndpointTest
-Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.937 sec - in it.io.openliberty.guides.system.SystemEndpointTest
-Running it.io.openliberty.guides.inventory.InventoryEndpointTest
-Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.396 sec - in it.io.openliberty.guides.inventory.InventoryEndpointTest
-Running it.io.openliberty.guides.faulttolerance.FaultToleranceTest
-Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 3.517 sec - in it.io.openliberty.guides.faulttolerance.FaultToleranceTest
+Running it.io.openliberty.guides.system.SystemEndpointIT
+Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.937 sec - in it.io.openliberty.guides.system.SystemEndpointIT
+Running it.io.openliberty.guides.inventory.InventoryEndpointIT
+Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.396 sec - in it.io.openliberty.guides.inventory.InventoryEndpointIT
+Running it.io.openliberty.guides.faulttolerance.FaultToleranceIT
+Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 3.517 sec - in it.io.openliberty.guides.faulttolerance.FaultToleranceIT
 
 Results :
 
 Tests run: 3, Failures: 0, Errors: 0, Skipped: 0
 ----
 
 To see if the tests detect a failure, comment out the [hotspot=40-41 hotspot=51-52]`changeSystemProperty()` methods
-in the [hotspot]`FaultToleranceTest.java` file. Rerun the Maven build.
-You see a test failure occur for the [hotspot=34-53]`testFallbackForGet()` test case.
+in the [hotspot]`FaultToleranceIT.java` file. Rerun the tests to see a test failure occur for the [hotspot=34-54]`testFallbackForGet()` test case.
```

Comment:
Change to:
"To see whether the tests detect a failure, comment out the `changeSystemProperty()` methods in the `FaultToleranceIT.java` file. Rerun the tests to see that a test failure occurs for the `testFallbackForGet()` test case."

Reply:
Updated